docs/installation/apt-download-deb.md

```diff
@@ -0,0 +1,49 @@
+# Install Percona Server for MySQL 8.0 using downloaded DEB packages
+
+Download the packages of the desired series for your architecture from the
+[Percona downloads](http://www.percona.com/downloads/Percona-Server-8.0/) page. The easiest way is to download the bundle which contains
+all the packages. The following example downloads [Percona Server for MySQL 8.0.13-3](../release-notes/Percona-Server-8.0.13-3.md)
+
+
+ release packages for Debian 9.0 (stretch):
```

Comment:
This phrase is detached from the sentence

Reply:
fixed.